history-of-the-quakers-v3/original/33-book-6-chapter-13.adoc

```diff
@@ -1001,8 +1029,11 @@ made it an occurrence much taken notice of; and was the more extraordinary,
 that he does not appear to have been a man apt to be carried away by undue transports,
 but on the contrary was much favoured in his public services.
 
-At the ferry,`" says Thomas Story,
-"`I had the afflicting news of the death of my companion Roger Gill at Philadelphia,
+[.embedded-content-document.testimony]
+--
+
+At the ferry, says Thomas Story,
```

Comment:
so this is offset in the original, but I wasn't sure how to embed it here with the part right at the beginning that says "says Thomas Story" without have the rest of it in quotation marks....does that make sense?? Like it's not truly a direct quote with that in there, but I didn't want to use any quotation marks since its embedded. I hope that makes sense! Maybe I'm being too technical or picky, idk! ☺️

Reply:
yup, i know what you mean, let's change it to this:

```ASCIIDOC
At the ferry, [says Thomas Story,]
```